services/user.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth import get_user_model
+from typing import Any
+
+
+User = get_user_model()

```

Comment:
You should use `get_user_model()` directly instead of assigning it to `User`. This is important because the user model can be swapped out with a custom user model. Here is how you should do it:
python
get_user_model().objects.create_user(username=username, password=password)
```
This is a code quality issue.